clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/data/Keys.groovy

```diff
@@ -101,16 +107,32 @@ class Keys {
   }
 
   static String getLoadBalancerKey(String loadBalancerName, String account, String region, String vpcId, String loadBalancerType) {
+    return getLoadBalancerKey(loadBalancerName, account, region, vpcId, loadBalancerType, null)
+  }
+
+  static String getLoadBalancerKey(String loadBalancerName, String account, String region, String vpcId, String loadBalancerType, String loadBalancerArn) {
     String key = "${ID}:${Namespace.LOAD_BALANCERS}:${account}:${region}:${loadBalancerName}"
     if (vpcId) {
       key += ":${vpcId}"
       if (loadBalancerType && loadBalancerType != 'classic') {
         key += ":${loadBalancerType}"
+        if (loadBalancerArn != null) {
```

Comment:
is it legitimate to have a non-classic ELB without an ARN?

is there something that needs the ARN to be adde to the key? (can't we infer it by load balancer name)